internal/middleware/logger.go

```diff
@@ -31,6 +30,9 @@ func LoggerMiddleware(rootLogger *zerolog.Logger) func(next http.Handler) http.H
 				Logger()
 			t1 := time.Now()
 
+			lHeaders := logger.With().Fields(r.Header).Logger()
+			lHeaders.Debug().Msg("Started request with above headers")
```

Comment:
There is no need to copy logger just to use it once, why don't you just do:

```
logger.Debug().Fields(r.Header).Msgf("Started %s request %s", r.Method, r.URL.Path")
```

I am asking once again for the same formatting as we do for the "Completed" message so I can "pair" the two by eyes in Kibana.

Reply:
I do not know how to create a submap, I think this is okay.

Reply:
oh I missed this comment, there is usage of method on a pointer and for some reason it doesn't work in-line (no idea what went wrong there, didn't investigate).